### DISCUSSION TITLE
Is there a way to delete a font from the Character Map UWP?

### DISCUSSION THREAD

**Discussion by D0nna4506** [character-map-uwp/Character-Map-UWP#299] [Q&A] [answered]
I would like to know if there is a way to delete a font from the Character Map UWP.

**Comment by JohnnyWestlake** [comment, marked as answer]
> I would like to know if there is a way to delete a font from the Character Map UWP.

Hi, thanks for reaching out;

By default Character Map UWP shows all fonts installed in Windows. If the font is installed to Windows, you can uninstall the font through Windows Settings, which you can access from Character Map UWP by the following:
- Open Settings in the top right
- Choose "Font Management" on the left
- Chose "Manage Installed Fonts"

This will open Windows Settings font management page. From here you can select a font and uninstall it from Windows.


If you have imported the font into Character Map UWP yourself, you can right click the font in the main font list on the left hand side of the app and select "Delete Font" to remove it.

If you do not want to uninstall a font from Windows but would like to hide it from Character Map UWP, that option currently doesn't exist, but you can create collections of Fonts instead if you'd like if you only want to see certain fonts.
